src/main/java/format/FormatType.java

```diff
@@ -0,0 +1,10 @@
+package format;
+
+public enum FormatType {
+    XML,
+    JSON;
+
+    public String getXmlHeader(){
```

Comment:
xml function in enum that contains JSON format ?

you need different class for XMLUtils